relay-server/src/services/processor/event.rs

```diff
@@ -68,12 +69,9 @@ pub fn extract<G: EventProcessing>(
         return Err(ProcessingError::DuplicateItem(duplicate.ty().clone()));
     }
 
-    let is_normalization_enabled =
-        if config.processing_enabled() && global_config.options.processing_disable_normalization {
-            false
-        } else {
-            config.normalization_level().is_enabled()
-        };
+    let normalization_skipped = config.processing_enabled()
```

Comment:
nit: The skipping has not happened yet at this point.

```suggestion
    let skip_normalization = config.processing_enabled()
```